apps/expo/package.json

```diff
@@ -43,6 +43,7 @@
     "react-native-root-toast": "^3.5.1",
     "react-native-safe-area-context": "4.6.3",
     "react-native-screens": "~3.22.1",
+    "react-native-svg": "13.9.0",
```

Comment:
is this still being used?

Reply:
No, but should I leave it for other to use?

Reply:
you can revert it, and someone else can add it if they're using it